README.md

```diff
@@ -10,6 +10,32 @@ CircleCI 2.1 introduces [several new features](https://github.com/CircleCI-Publi
 
 For more info on Orbs, checkout their [docs](https://github.com/CircleCI-Public/config-preview-sdk/tree/master/docs)!
 
+The TL;DR is that an `orb` yml configuration file that's used to share things like [executors](orb-executors), [commands](orb-commands), and [jobs](orb-jobs) across your CircleCi builds.
+
+## Getting Started
+
+All our orbs are `.yml` files that are stored in `src/<orb-name>/<orb-name>.yml`. The nested directory is so that every orb can have associated documentation beside it.
```

Comment:
```suggestion
Orb files are stored in `src/<orb-name>/<orb-name>.yml`. The nested directory is so that every orb can have associated documentation beside it.
```

Reply:
(Removed the duplicate `.yml` explanation since it's in the first paragraph)